components/infobox/wikis/stormgate/infobox_skill_custom.lua

```diff
@@ -15,19 +15,22 @@ local Lua = require('Module:Lua')
 local Page = require('Module:Page')
 local String = require('Module:StringUtils')
 local Table = require('Module:Table')
+local messageBox = require('Module:Message box')
```

Comment:
uppercase first for module imports please

Reply:
done